src/app/dashboard/dashboard.html

```diff
@@ -0,0 +1,107 @@
+<div id="userbar">
+	<img src="" />
+	<a href="">Log Out</a>
+</div>
+
+<ul class="stocklist">
+<li class="ticker">
```

Comment:
awesome, now that you have this setup, move the ticker symbol names in the controller `this.stocks = [1];` and use an ng-repeat on the list item:
```
    <li class="ticker" ng-repeat="stock in dash.stocks">
        <h1>^{{stock}}</h1>
        <div class="nodules">
            <span class="good"></span>
            <span class="bad"></span>
            <span class="bad"></span>
            <span class="neutral"></span>
            <span class="good"></span>
        <h2>Hold</h2>
    </li>
```

then you can only have one li item in your html that will get dynamically populated and you wont need any other list items

so in your controller your stocks will look something like this:
```javascript
    this.stocks = [
        'SNP',
        'NASDAQ',
        'DJI',
        'STFU'
    ];
```